spec/features/listing_all_cheets_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+
+  feature "User can browse the list of cheets" do
+
+    before(:each){
```

Comment:
You can merge the two before blocks and put everything in one block, you also shouldn't use {} for multi line blocks